frontend/src/components/Field/Attachment/Attachment.tsx

```diff
@@ -339,12 +339,7 @@ export const Attachment = forwardRef<AttachmentProps, 'div'>(
               textStyle="body-2"
               aria-hidden
             >
-              {t(
-                'features.adminForm.sidebar.fields.imageAttachment.maxFileSize',
-                {
-                  readableMaxSize,
-                },
-              )}
+              Maximum file size: {readableMaxSize}
```

Comment:
Hang on, isn't this a reversion?

Reply:
It's indeed a reversion. Reason being, the UI Tests on Storybook/Chromatic kept showing `{readableMaxSize}`, rather than replacing it as such:

<img width="848" alt="image" src="https://github.com/user-attachments/assets/ce185856-202c-4b00-a970-90caf6b0a4db">

Unfortunately, this value is used in many of the UI test pages, resulting in quite a number of UI diff issues raised by Chromatic.

To reduce the workload for the code review, I reverted it so we can push most of the generally non-problematic translations, as well as the new file structure for the translations. This would allow others to work on the new structure as well. After which we could make a more incremental PR for this problem.

What are your thoughts on this? Should I not have reverted it and let the team take a look?

Reply:
I think that's fine then.